docker/publish-public-images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -ex
+
+project=$1
+
+make -C ../hail python/hail/hail_pip_version
+cp ../hail/python/hail/hail_pip_version hail/hail_pip_version
+hail_pip_version=$(cat hail/hail_pip_version)
+
+build_and_push() {
+    name=$1
+    base=$2
+
+    docker=hailgenetics/$name:$hail_pip_version
```

Comment:
Did you decide not to tag these with the SHA anymore?

Reply:
These images were never tagged with a SHA, they're tagged with the pip version because their version of hail is from PyPI e.g. https://hub.docker.com/r/hailgenetics/genetics/tags?page=1&ordering=last_updated

Reply:
Sorry. I should have been more clear. We used to have this as well: `GENETICS_PUBLIC_IMAGE = gcr.io/$(PROJECT)/genetics-public:$(shell docker images -q --no-trunc genetics-public:latest | sed -e 's,[^:]*:,,')`